### PR TITLE
Non pri cef fix

### PR DESCRIPTION
**Pull request by cwadhwani-splunk**
Updated the regex for non pri cef formated logs such that a '<space>digit' date will be accepted.
Ticket link: https://splunk.atlassian.net/browse/ADDON-76507

### CODE REVIEW

File: package/etc/conf.d/conflib/raw/app-raw-bsd_nopri.conf [hunk 1]
```diff
@@ -53,7 +53,7 @@ block parser app-raw-bsd_nopri() {
 };
 application app-raw-bsd_nopri[sc4s-raw-syslog] {
     filter {
-        message('^\w\w\w \d\d \d\d:\d\d:\d\d ');
+        message('^\w\w\w [ \d]\d \d\d:\d\d:\d\d ');
```
Comment (rjha-splunk) [inline package/etc/conf.d/conflib/raw/app-raw-bsd_nopri.conf:56]: Can we make it better regex as well ?

File: package/lite/etc/conf.d/conflib/raw/app-raw-bsd_nopri.conf [hunk 2]
```diff
@@ -53,7 +53,7 @@ block parser app-raw-bsd_nopri() {
 };
 application app-raw-bsd_nopri[sc4s-raw-syslog] {
     filter {
-        message('^\w\w\w \d\d \d\d:\d\d:\d\d ');
+        message('^\w\w\w [ \d]\d \d\d:\d\d:\d\d ');
```
Comment (rjha-splunk) [inline package/lite/etc/conf.d/conflib/raw/app-raw-bsd_nopri.conf:56]: Same comment

**Review by rjha-splunk — COMMENTED** [review]
Please try to make better regex for dates what i mean by that the dates should be between 1-31 and may be for month as well.

File: package/etc/conf.d/conflib/raw/app-raw-bsd_nopri.conf (hunk 1 above)
Comment (cwadhwani-splunk) [reply package/etc/conf.d/conflib/raw/app-raw-bsd_nopri.conf]: Updated the regex with some more constraints.

File: package/lite/etc/conf.d/conflib/raw/app-raw-bsd_nopri.conf (hunk 2 above)
Comment (cwadhwani-splunk) [reply package/lite/etc/conf.d/conflib/raw/app-raw-bsd_nopri.conf]: Updated the regex with some more constraints.

**Pull request merged**